crates/bevy_plugin/tests/utils/mod.rs

```diff
@@ -32,10 +32,6 @@ pub trait AppExt {
     fn dialogue_runner(&mut self) -> &DialogueRunner;
     #[must_use]
     fn dialogue_runner_mut(&mut self) -> Mut<DialogueRunner>;
```

Comment:
How do we decide what goes on the AppExt v the WorldExt? It seems like a lot these fns only use world access?

Reply:
Honestly, that was completely arbitrary to fix a lifetime issue. I just noticed the code was unused anyways, so I removed the functions in `WorldExt` and `WorldExt` itself entirely :)